R/html_resources.R

```diff
@@ -269,8 +277,7 @@ discover_rmd_resources <- function(rmd_file, encoding,
               recursive = TRUE,
               include.dirs = FALSE)
             lapply(files, function(f) {
-              list(path = file.path(explicit_res$path, f), 
-                   explicit = TRUE,
+              discover_single_resource(file.path(explicit_res$path, f), TRUE,
```

Comment:
All that said, I can't convince myself that the glob paths are added to any resource list. What happens to that list of named lists that is created? The output of the `lapply` was previously assigned to `resources`, but that doesn't appear to be used...